sycl/include/sycl/ext/oneapi/experimental/bfloat16.hpp

```diff
@@ -14,7 +14,7 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace ext {
-namespace intel {
+namespace oneapi {
 namespace experimental {
 
 class [[sycl_detail::uses_aspects(ext_intel_bf16_conversion)]] bfloat16 {
```

Comment:
This should be wrapped with SYCL_DEVICE_ONLY defined, for it to be built successfully by other compilers, right? (see https://github.com/intel/llvm/pull/5594/)
@AlexeySotkin, are you planning for adding this as a separate PR?